infra-cdk/lib/stacks/ReportsBucketStack.ts

```diff
@@ -0,0 +1,32 @@
+import * as cdk from 'aws-cdk-lib';
+import * as s3 from 'aws-cdk-lib/aws-s3';
+import { Construct } from 'constructs';
+import { Stage, Tenant } from '../core/enums';
+import { BaseStackProps } from '../core/types';
+
+const BASE_TABLE_NAME = 'S3Bucket';
+
+export class S3BucketStack extends cdk.Stack{
```

Comment:
The class name should match the file name, so please call this class `ReportsBucketStack` too